test/e2e/vl-core.test.js

```diff
@@ -0,0 +1,17 @@
+const {assert, driver} = require('vl-ui-core').Test.Setup;
+const VlCorePage = require('./pages/vl-core.page');
+
+
+describe('vl-core', async () => {
+  const vlCorePage = new VlCorePage(driver);
+
+  before(() => {
+    return vlCorePage.load();
+  });
+
+  it('visible', async () => {
+    const slowElement = await vlCorePage.getSlowElement();
+    await assert.eventually.isFalse(slowElement.isDisplayed());
+    await driver.wait(async () => await slowElement.isDisplayed(), 3000);
```

Comment:
zeker dat uw test dan voldoende dekt dat het pas displayed wordt when defined? kan niet aan selenium zijnen isDisplayed liggen nu?

Reply:
De eerste assert checkt op not displayed, dus lijkt me OK zo?

Reply:
ja, misschien zijde daar "te rap op een andere manier dan nog niet defined", als ge begrijpt wat ik bedoel.

Reply:
Ja, misschien dan best in de wait functie blijven assert uitvoeren dat het niet displayed is en bij de catch (timeout) of erbuiten de displayed controleren?

Reply:
k weet het nie he, tis een vraag. ge hebt da gedrag toch ook zelf eens "gezien" @wautergu ?

Reply:
ligt het flikkeren niet eerder soms aan het feit dat het element al gedefinieerd is maar zijne stijl nog niet?

Reply:
In kader van Zendantenneskaart gebruiken we de .min variant, dus zit de stijl inline. Ik denk dat de oorzaak hier eerder is dat er te veel (zware) logica zit in de connectedcallback waardoor het renderen geblokkeerd wordt. Verder te bekijken in [#38](https://github.com/milieuinfo/webcomponent-vl-ui-side-sheet/issues/38).

Reply:
ge bedoelt, "dat het geen goed idee was om die hele kaart in ene keer op te halen" toch nie zeker :-)

Reply:
> ge bedoelt, "dat het geen goed idee was om die hele kaart in ene keer op te halen" toch nie zeker :-)

Nee, dat heeft er zelfs totaal niets mee te maken. Het probleem is geïsoleerd te reproduceren bij vl-ui-side-sheet.

Reply:
> k weet het nie he, tis een vraag. ge hebt da gedrag toch ook zelf eens "gezien" @wautergu ?

Bij de zendkaart? Ja